src/main/java/org/ladocuploader/app/upload/S3ReadOnlyCloudFileRepository.java

```diff
@@ -0,0 +1,59 @@
+package org.ladocuploader.app.upload;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.amazonaws.services.s3.transfer.Download;
+import com.amazonaws.services.s3.transfer.TransferManager;
+import com.amazonaws.services.s3.transfer.TransferManagerBuilder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+
+@Service
+@Profile("!test")
+@Slf4j
+public class S3ReadOnlyCloudFileRepository implements ReadOnlyCloudFileRepository {
```

Comment:
I think we only needed this class bc FFB didn't support "download" - it looks like it does now though as "get". Can we use that instead?

Reply:
Yes, good point! I am using it now